table-api/src/main/java/io/deephaven/api/Selectable.java

```diff
@@ -0,0 +1,36 @@
+package io.deephaven.api;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public interface Selectable {
+
+    static Selectable parse(String x) {
+        if (ColumnName.isValidParsedColumnName(x)) {
+            return ColumnName.parse(x);
+        }
+        // note: we are unable to parse a selectable into a strongly typed RHS
+        // (either ColumnAssignment, or ColumnFormula with strong RHS).
```

Comment:
I sort of understand this comment, but it raises more questions than it answers.